docs/RaidbossGuide.md

```diff
@@ -44,6 +44,11 @@ Prefer using this over zoneRegex.
 A trigger set must have one of zoneId or zoneRegex to specify the zone
 (but not both).
 
+**initData**
+A function that can be used to initialize the data this trigger set uses.
+This function is called any time the fight is reset, mainly on zone change or wipe.
```

Comment:
```suggestion
It should return an object that sets values for any fields in `data` that need to be initialized.
This function is called any time the fight is reset, mainly on zone change or wipe.
```

Reply:
I think to add a type hint makes sense?
i.e. something like:

> **initData**
> `(data: RaidbossData) => Data`
> blahblah

Reply:
In general I think I would agree with you, but in this specific case I worry that `() => Omit<Data, keyof RaidbossData>` might be more confusing than helpful.